src/Microsoft.AspNetCore.Server.Kestrel/Networking/Libuv.cs

```diff
@@ -610,6 +674,9 @@ private static class NativeDarwinMonoMethods
             public static extern void uv_unref(UvHandle handle);
 
             [DllImport("__Internal", CallingConvention = CallingConvention.Cdecl)]
+            public static extern int uv_fileno(UvHandle handle, ref IntPtr socket);
```

Comment:
If they need it they would have to add them. I don't really care too much - the `__Internal` is going away soon. It won't/can't work with dotnet and even [now](https://github.com/aspnet/KestrelHttpServer/commit/fe75aa47dbe4209600511543fb5b2f9aca055c8a) when using dotnet the regular DllImports are being used on Mac on Mono.